test/testobjects/testentitybatches.ts

```diff
@@ -52,7 +52,7 @@ export class TestEntityBatches {
 
     public static GetCounterBatch(operations: CounterOperation[], existingState: number | undefined): EntityInputsAndOutputs {
         const id = new df.EntityId("stringstore", "stringstorekey");
-        let currentState: number;
+        let currentState: number | null = null;
```

Comment:
What does the `null = null` expression mean?

Reply:
This would be more clear as `let currentState: (number | null) = null;`. The first null is a part of the type signature. That being said, I should probably use `undefined` to follow the rest of the codebase.